etc/config/c++.amazon.properties

```diff
@@ -2911,6 +2911,13 @@ libs.abseil.staticliblink=absl_bad_any_cast_impl:absl_failure_signal_handler:abs
 libs.abseil.versions.trunk.version=trunk
 libs.abseil.versions.trunk.path=/opt/compiler-explorer/libs/abseil
 
+libs.array.name=Array
+libs.array.versions=trunk
+libs.array.url=https://github.com/dsharlet/array
+libs.array.description=A multidimensional array class for C++, with optional compile-time parameters
+libs.array.versions.trunk.version=trunk
+libs.array.versions.trunk.path=/opt/compiler-explorer/libs/array/master/include
```

Comment:
As noted in the infra PR this is fine but will be a one-time snapshot of whatever `array`'s master is when I run the installer. (no changes needed here to change that behaviour, just noting here to make sure things are surfaced :))